src/thor/astar_bss.cc

```diff
@@ -29,9 +29,7 @@ constexpr uint32_t kMaxIterationsWithoutConvergence = 200000;
 AStarBSSAlgorithm::AStarBSSAlgorithm(const boost::property_tree::ptree& config)
     : PathAlgorithm(config.get<uint32_t>("max_reserved_labels_count_astar",
                                          kInitialEdgeLabelCountAstar),
-                    config.get<bool>("clear_reserved_memory", false)),
-      max_label_count_(std::numeric_limits<uint32_t>::max()), mode_(travel_mode_t::kDrive),
```

Comment:
this is a frequent change in this PR, `max_label_count_` must've been used at some point as termination criteria and was copy/pasted along. however, it's not used anywhere

Reply:
Maybe I don't understand smth but it was used here https://github.com/valhalla/valhalla/pull/4372/files/c322b77e3eab02d1b60a0dee449be36575ac8a22#diff-acd5a91be66009c2563642ad11eed627f663a49e0fab12b2ec2c8fbce77c1702L292

Reply:
right, I should've said "not being effectively used anywhere". you can see your highlighted section is removed too, there was no point for this variable